docs/api.rst

```diff
@@ -192,20 +192,10 @@ automate
 Constants
```

Comment:
I'll let @spencerahill weigh in, but I think since we are denoting `_constants.py` as private API now, we should probably delete this section from the documentation entirely.

Reply:
I agree.